ui/src/plugins/com.google.PerfettoMcp/index.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2025 The Android Open Source Project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Trace} from '../../public/trace';
+import {App} from '../../public/app';
+import {PerfettoPlugin} from '../../public/plugin';
+import {McpServer} from '@modelcontextprotocol/sdk/server/mcp.js';
+import {Client} from '@modelcontextprotocol/sdk/client/index.js';
+import {InMemoryTransport} from '@modelcontextprotocol/sdk/inMemory.js';
+import {
+  CallableTool,
+  FunctionCallingConfigMode,
+  GoogleGenAI,
+  mcpToTool,
+} from '@google/genai';
+import {registerTraceTools} from './tracetools';
+import {z} from 'zod';
+import {Setting} from 'src/public/settings';
+import {ChatPage} from './chat_page';
+import m from 'mithril';
+export default class PerfettoMcpPlugin implements PerfettoPlugin {
+  static readonly id = 'com.google.PerfettoMcp';
```

Comment:
A few words describing your plugin I think could go a long way:

```
static readonly description = 'This plugin does ....';
```

This description shows up on the plugins page.